### PR TITLE
Create Book model

### PR DESCRIPTION
**Pull request by sadatetchamouza**
- Create Book model based on what it is done in the post "building note-taking app with Flutter"
- This is straightforward model : We just store the title and the cover image as string. And that's all we need now.

### CODE REVIEW

File: lib/model/book.dart [hunk 1]
```diff
@@ -0,0 +1,35 @@
+
+import 'package:cloud_firestore/cloud_firestore.dart';
+
+class Book {
+  final String id;
+  String title;
+  String cover;
+  final DateTime createdAt;
+  DateTime modifiedAt;
```
Comment (rygelouv) [inline lib/model/book.dart:9]: Why are some fields final and others not?

**Review by rygelouv — APPROVED** [review]
Looking great! Left a question

File: lib/model/book.dart (hunk 1 above)
Comment (sadatetchamouza) [reply lib/model/book.dart:9]: To be honest I just follow the post to see how he did it.  I guess final fields is those fields we don't want to modify later.

Comment (rygelouv) [reply lib/model/book.dart:9]: Alright. Sounds good

**Pull request merged**